### PR TITLE
Frontend fixes

### PR DESCRIPTION
**Pull request by Haferkorn**
- Background Images
- Login Redirection

### CODE REVIEW

File: frontend/src/components/pages/recommandations/Recommendations.js [hunk 1]
```diff
@@ -1,11 +1,12 @@
...
+import WineBackground from "./recommendations.png"
 
 function Recommendations({ recommendations }) {
    return (
-      <div>
-         <Heading>Your Recommendations:</Heading>
+      <BackgroundWrapper>
+
          <RecommendationContainer>
             {recommendations.length === 0 ? (
```
Comment (macben888) [inline frontend/src/components/pages/recommandations/Recommendations.js:11]:
```suggestion
            {!recommendations.length ? (
```

**Review by macben888 — APPROVED** [review]

File: frontend/src/components/pages/homepage/Homepage.js [hunk 2]
```diff
@@ -19,7 +21,7 @@ function Homepage() {
                autoplay
                loop
                src={WineAnimation}
-               style={{ height: "200px", width: "300px" }}>
+               style={{ height: "200px", width: "300px" ,padding:"0px 50px 0px 50px"}}>
```
Comment (christophersiem) [inline frontend/src/components/pages/homepage/Homepage.js:24]: if you only give 2 values, the first one is for `top` and `bottom`, the second for `left` and `right`

```suggestion
               style={{ height: "200px", width: "300px" ,padding:"0px 50px"}}>
```

File: frontend/src/components/pages/homepage/Homepage.js [hunk 3]
```diff
@@ -29,28 +31,53 @@ function Homepage() {
...
+  padding-top: 30px;
 `
 
+
 const Emphasis = styled.em`
    text-decoration: underline;
 `
-export const AnimationWrapper = styled.div``
+export const AnimationWrapper = styled.div`
+`
```
Comment (christophersiem) [inline frontend/src/components/pages/homepage/Homepage.js:75]:
```suggestion

```

**Review by christophersiem — APPROVED** [review]
🙌🏽

**Pull request merged**